slides/index.html

```diff
@@ -258,6 +258,49 @@
 ---
 class: center, middle
 
+# LF oppgave 2
+
+```cpp
+puck->addCharacteristic(
+  SAMPLE_GATT_SERVICE,
+  SAMPLE_GATT_CHARACTERISTIC, 20,
+  GattCharacteristic::
```

Comment:
I think being more explicit is better here. That way people have to think about what they send in, and not rely on the defaults and forget what they do

On Mon, Mar 2, 2015 at 2:09 AM, Sigve Sebastian Farstad
<notifications@github.com> wrote:

>> @@ -258,6 +258,49 @@
>>  ---
>>  class: center, middle
>>  
>> +# LF oppgave 2
>> +
>> +```cpp
>> +puck->addCharacteristic(
>> +  SAMPLE_GATT_SERVICE,
>> +  SAMPLE_GATT_CHARACTERISTIC, 20,
>> +  GattCharacteristic::
> This parameter defaults to this value, so it can be omitted if you want. I'm not sure what's best in terms of teaching.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/bftj/bftj.github.io/pull/83/files#r25574548